### ISSUE TITLE
support Angular2 or later

### ISSUE THREAD

**Issue by JianhuisHuang** [danialfarid/ng-file-upload#2117]
this plugin support Angular2 or later? Thanks. @danialfarid

**Comment by Sidd27** [comment]
@JianhuisHuang I don't think this plugin supports angular 2 and above you can try below plugin https://www.npmjs.com/package/ng2-file-upload

**Comment by JianhuisHuang** [comment]
> @JianhuisHuang I don't think this plugin supports angular 2 and above you can try below plugin https://www.npmjs.com/package/ng2-file-upload

**Comment by JianhuisHuang** [comment]
@Sidd27  thanks, I will try.

**Comment by markb-trustifi** [comment]
> @JianhuisHuang I don't think this plugin supports angular 2 and above you can try below plugin https://www.npmjs.com/package/ng2-file-upload

It doesn't support multi-part upload.
